<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 33, 33);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 33, 33);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>68</y>
       <width>611</width>
       <height>341</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>10000</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <row>
      <property name="text">
       <string>00</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Từ vựng</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Phiên âm</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Loại từ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Dịch nghĩa</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Đánh giá</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>hi</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>505</width>
       <height>52</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
</string>
     </property>
     <property name="title">
      <string>Menu</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sắp xếp theo:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="type_sort">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Mặc định</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bảng chữ cái</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tăng dần(%)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Giảm dần(%)</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Học từ:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="start_point">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Tới từ:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="end_point">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="default_">
     <property name="geometry">
      <rect>
       <x>521</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#default_{
border-radius: 6px;
background-color:rgb(85, 170, 255) ;
}
QPushButton#default_:hover {
  color:rgb(55, 165, 0)
}

QPushButton#default_:pressed {
  background-color:rgb(85, 255, 127);
}</string>
     </property>
     <property name="text">
      <string>Mặc định</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>521</x>
       <y>39</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#000000;&quot;&gt;Làm mới thiết lập&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#save{
border-radius: 6px;
background-color:rgb(85, 170, 255) ;
}
QPushButton#save:hover {
  color:rgb(55, 165, 0)
}

QPushButton#save:pressed {
  background-color:rgb(85, 255, 127);
}</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_row">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>420</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#000000;&quot;&gt;Thêm hàng mới&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#add_row{
border-radius: 10px;
background-color:rgb(85, 170, 255) ;
}
QPushButton#add_row:hover {
  color:rgb(55, 165, 0)
}

QPushButton#add_row:pressed {
  background-color:rgb(85, 255, 127);
}</string>
     </property>
     <property name="text">
      <string>Add row</string>
     </property>
    </widget>
    <widget class="QPushButton" name="remove_row">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>420</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#000000;&quot;&gt;Xoá hàng hiện tại&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#remove_row{
border-radius: 10px;
background-color:rgb(85, 170, 255) ;
}
QPushButton#remove_row:hover {
  color:rgb(55, 165, 0)
}

QPushButton#remove_row:pressed {
  background-color:rgb(85, 255, 127);
}</string>
     </property>
     <property name="text">
      <string>Remove row</string>
     </property>
    </widget>
    <widget class="QPushButton" name="learing">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>420</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#000000;&quot;&gt;Bắt đầu học&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#learing{
border-radius: 10px;
background-color:rgb(85, 170, 255) ;
}
QPushButton#learing:hover {
  color:rgb(55, 165, 0)
}

QPushButton#learing:pressed {
  background-color:rgb(234, 0, 0)
}</string>
     </property>
     <property name="text">
      <string>Learing</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="home">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>410</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
